### PR TITLE
Fix style issues

### PR DESCRIPTION
**Pull request by Luyao-Zhang-1**

### CODE REVIEW

File: content/refguide7/app-settings-dialog.md [hunk 1]
```diff
@@ -17,18 +17,16 @@ These settings are optional. You can click **Create app** to create your app wit
...
+The Mendix Platform offers online services such as [version control](version-control), cloud deployment, and collaboration. If you select **Yes** for **Enable online services**, this will create a project in the Developer Portal and a corresponding version control repository. If you choose **No** for **Enable online services**, you will create an app that is only stored on your local disk. At a later point you can still decide to upload this local app to a version control server and enjoy the benefits of version control.
 
 ## 4 Default Language
 
-The default language is the language of the user interface of your app. Choose the language that you will initially use in your forms and other user interface elements. You can always add additional languages to your app later.
+From the **Default language** down-down list, select the language of the user interface of your app. Choose the language that you will initially use in your forms and other user interface elements. You can always add additional languages to your app later.
```
Comment (Adam-Dupaski) [inline content/refguide7/app-settings-dialog.md:28]: "down-down"

File: content/refguide7/app-settings-dialog.md [hunk 2]
```diff
@@ -17,18 +17,16 @@ These settings are optional. You can click **Create app** to create your app wit
...
 
 ## 3 Enable Online Services
 
-The Mendix Platform offers online services such as [version control](version-control), cloud deployment and collaboration. When enabled, this will create a project in the Developer Portal and a corresponding version control repository.
-
-If you choose **No** for **Enable online services**, you will create an app that is only stored on your local disk. At a later point you can still decide to upload this local app to a version control server and enjoy the benefits of version control.
+The Mendix Platform offers online services such as [version control](version-control), cloud deployment, and collaboration. If you select **Yes** for **Enable online services**, this will create a project in the Developer Portal and a corresponding version control repository. If you choose **No** for **Enable online services**, you will create an app that is only stored on your local disk. At a later point you can still decide to upload this local app to a version control server and enjoy the benefits of version control.
```
Comment (Adam-Dupaski) [inline content/refguide7/app-settings-dialog.md:24]: Probably don't need second instance of **Enable online services**

File: content/refguide7/download-from-version-control-dialog.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ In the **App repository address** field, enter the repository address of the app
 
 {{% alert type="info" %}}
 
-This option is only available when support for other servers is enabled in **Edit** > **Preferences** > **Advanced** > [Enable private version control](preferences-dialog#enable).
+This option is only available when support for other servers is enabled in the [Preferences dialog box](preferences-dialog#enable).
```
Comment (Adam-Dupaski) [inline content/refguide7/download-from-version-control-dialog.md:45]: Please look for other examples of cross-references to see if the UI element - "dialog box" - should be in the link text

File: content/refguide7/export-project-package-dialog.md [hunk 4]
```diff
@@ -43,4 +45,4 @@ This option becomes available after you run the app locally at least once, becau
 
 ## 4 Read More
 
-* [Import Project Package dialog box](import-project-package-dialog)
+* [Import Project Package Dialog](import-project-package-dialog)
```
Comment (Adam-Dupaski) [inline content/refguide7/export-project-package-dialog.md:48]: Is this the correct title of the doc?

File: content/refguide7/import-project-package-dialog.md [hunk 5]
```diff
@@ -12,39 +12,39 @@ To open this dialog box,  go to **File > Import App Package**, browse to the *.m
...
 
 #### 2.1.1 New Mendix Team Server
 
-If you select to store your app in a new Mendix Team Server, a new Team Server app will be created. You need to enter the name for the new Team Server app and repository in the **App name** field. 
+If you select this option, a new Team Server repository will be created to store the app. You need to enter the name for the new Team Server app and repository in the **App name** field. 
 
 #### 2.1.2 Existing Mendix Team Server
 
-If you want to use an existing repository, select the app in the **Team Server App** option.
+If you select this option, the app will be uploaded to an existing Team Server repository. You need to select the repository in the **Team Server App** drop-down list.
```
Comment (Adam-Dupaski) [inline content/refguide7/import-project-package-dialog.md:27]: "drop-down list" or "drop-down menu"?

File: content/refguide7/import-project-package-dialog.md [hunk 6]
```diff
@@ -12,39 +12,39 @@ To open this dialog box,  go to **File > Import App Package**, browse to the *.m
...
-### 2.2 Private Server
+### 2.2 Private Server {#private-server}
 
-In the **App repository address** field, enter the address of the repository you want to upload your app to.
+If you select this option, the app will be stored on a private server. You need to enter the **App repository address** to which you want to upload your app.
 
 {{% alert type="info" %}}
 
-This option is only available when support for other servers is enabled in **Edit** > **Preferences** > **Advanced** > [Enable private version control](preferences-dialog#enable).
+This option is only available when support for other servers is enabled in the [Preferences dialog box](preferences-dialog#enable).
```
Comment (Adam-Dupaski) [inline content/refguide7/import-project-package-dialog.md:41]: Please review.

File: content/refguide7/import-project-package-dialog.md [hunk 7]
```diff
@@ -54,8 +54,8 @@ Select this option if you do not need to upload the new app to a version control
 
 ## 3 Disk Location
 
-Use this field to choose the directory where the project files of the app will be stored. If version control is enabled, the suggested name includes ends with **-main** to indicate this will be the main development line of the app.
+In the **Project directory** field, specify the directory where the project files of the app will be stored. If version control is enabled, the suggested name includes ends with **-main** to indicate this will be the main development line of the app.
 
 ## 4 Read More
 
-* [Export Project Packge dialog](export-project-package-dialog)
+* [Export Project Packge Dialog](export-project-package-dialog)
```
Comment (Adam-Dupaski) [inline content/refguide7/import-project-package-dialog.md:61]: Please review.

File: content/refguide7/open-app-dialog.md [hunk 8]
```diff
@@ -26,9 +26,9 @@ For more information about development lines, see [Version Control](version-cont
 
 In the **SVN repository address** field, enter the address of the app you want to open and press the **Connect** button to load the development lines from the repository. Then choose the development line in which you want to start developing.
 
-{{% alert type="warning" %}}
+{{% alert type="info" %}}
 
-The **Other SVN server** option is only available when support for other SVN servers is enabled in the [Preferences dialog box](preferences-dialog#enabled).
+The **Other SVN server** option is only available when support for other SVN servers is enabled in the [Preferences dialog box](preferences-dialog#enable).
```
Comment (Adam-Dupaski) [inline content/refguide7/open-app-dialog.md:31]: Please review link text.

File: content/refguide7/upload-to-version-control-dialog.md [hunk 9]
```diff
@@ -39,7 +39,7 @@ In the **App repository address** field, enter the repository address you want t
 
 {{% alert type="info" %}}
 
-This option is only available when support for other servers is enabled in **Edit** > **Preferences** > **Advanced** > [Enable private version control](preferences-dialog#enable).
+This option is only available when support for other servers is enabled in the [Preferences dialog box](preferences-dialog#enable).
```
Comment (Adam-Dupaski) [inline content/refguide7/upload-to-version-control-dialog.md:42]: Please review link text.

File: content/refguide7/import-project-package-dialog.md (hunk 5 above)
Comment (Luyao-Zhang-1) [reply content/refguide7/import-project-package-dialog.md:27]: Sorry for the typo. Should have checked more carefully. :pensive:

File: content/refguide7/app-settings-dialog.md (hunk 2 above)
Comment (Luyao-Zhang-1) [reply content/refguide7/app-settings-dialog.md:24]: 👍

File: content/refguide7/app-settings-dialog.md (hunk 1 above)
Comment (Adam-Dupaski) [reply content/refguide7/app-settings-dialog.md]: "drop-down menu" per SG

File: content/refguide7/download-from-version-control-dialog.md (hunk 3 above)
Comment (Adam-Dupaski) [reply content/refguide7/download-from-version-control-dialog.md:45]: "[Preferences](preferences-dialog#enable) dialog box."

Comment (Adam-Dupaski) [reply content/refguide7/download-from-version-control-dialog.md:45]: `"enabled via the [Enable Private Version Control with Subversion](preferences-dialog#enable) option in the **Preferences** dialog box."`

File: content/refguide7/export-project-package-dialog.md (hunk 4 above)
Comment (Adam-Dupaski) [reply content/refguide7/export-project-package-dialog.md:48]: Update title

File: content/refguide7/import-project-package-dialog.md (hunk 5 above)
Comment (Adam-Dupaski) [reply content/refguide7/import-project-package-dialog.md:27]: "drop-down menu" via SG

File: content/refguide7/import-project-package-dialog.md (hunk 6 above)
Comment (Adam-Dupaski) [reply content/refguide7/import-project-package-dialog.md:41]: Same as above

File: content/refguide7/import-project-package-dialog.md (hunk 7 above)
Comment (Adam-Dupaski) [reply content/refguide7/import-project-package-dialog.md:61]: Same as above

File: content/refguide7/open-app-dialog.md (hunk 8 above)
Comment (Adam-Dupaski) [reply content/refguide7/open-app-dialog.md:31]: Same as above

File: content/refguide7/upload-to-version-control-dialog.md (hunk 9 above)
Comment (Adam-Dupaski) [reply content/refguide7/upload-to-version-control-dialog.md:42]: Same as above

File: content/refguide7/import-project-package-dialog.md (hunk 5 above)
Comment (Luyao-Zhang-1) [reply content/refguide7/import-project-package-dialog.md:27]: Changed the text to "drop-down list" or "drop-down menu" according to the SG.

File: content/refguide7/consistency-errors-pages.md [hunk 10]
```diff
@@ -130,7 +130,7 @@ The most common errors for input widgets, their causes, and ways to fix them are
...
-| Select an attribute for this {widget name}.                  | You have added an input widget, and it is inside a data container, but an attribute is not selected for it. | Right-click the widget, click **Select Attribute** in the drop-down list, and set an attribute; or open widget's properties > the **Data source** section and set an attribute in the **Attribute (path)** field. |
+| Select an attribute for this {widget name}.                  | You have added an input widget, and it is inside a data container, but an attribute is not selected for it. | Right-click the widget, click **Select Attribute** in the drop-down menu, and set an attribute; or open widget's properties > the **Data source** section and set an attribute in the **Attribute (path)** field. |
```
Comment (Adam-Dupaski) [inline content/refguide7/consistency-errors-pages.md:133]: "Drop-down menu" and "drop-down list" used - that's okay, but is it necessary to differentiate these? Or would it be better to be consistent, using "list" or "menu" throughout a doc?

Comment (Luyao-Zhang-1) [reply content/refguide7/consistency-errors-pages.md:133]: Hi Adam, I can only find two instances, and they are both "drop-down menu". 🤔

Comment (Adam-Dupaski) [reply content/refguide7/consistency-errors-pages.md:133]: Yes, and ~19 instances of "drop-down list" - so is "drop-down menu" necessary?

**Pull request merged**